### ISSUE TITLE
author class with extra first

### ISSUE THREAD

**Issue by pirg** [pcooksey/bibtex-js#30]
Hi, I would like to print only the first author of each bibtex entry. I tried to add the extra="first" tag to the author class <span> 

```
<div class="if author">
   <span class="author" extra="first"></span>
</div>
```

but I still get the full author list.

**Comment by pirg** [comment]
Indeed, I would like to reduce very long (~ 10s-100s, eg: http://cdsads.u-strasbg.fr/abs/2014A%26A...571A..13P) author lists to "first author et al." to be able to output a more compact publication list.

**Comment by pcooksey** [comment]
That is a very long list of authors. How about an option to specify the max number of authors to be printed?

```html
<span class="author" max="1"></span>
```
And bibtex_js prints the `et al.` for 1 and `, et al.` for greater than 1 after the max author.

**Comment by pirg** [comment]
That's even better.
